### PR TITLE
add Angular Turkey

### PR DESCRIPTION
**Pull request by mobilerast**

### CODE REVIEW

File: src/assets/json/communities.json [hunk 1]
```diff
@@ -2051,4 +2051,23 @@
       }
     ]
   }
+  "Angular Turkey": { // Communities name
```
Comment (tonivj5) [inline src/assets/json/communities.json:2054]: Please, remove all comments :+1:

**Comment by Caballerog** [comment]
Have you a community image? ☺️

**Comment by mobilerast** [comment]
@Caballerog I could not able to upload in the images_raw dir. Here is the PNG version of our logo; https://avatars2.githubusercontent.com/u/47100904?s=200&v=4

**Comment by Caballerog** [comment]
@mobilerast don't worry, I've upload the image and edit the information in the JSON :-)

**Pull request merged**